content/blog/2017/0.1-canary/index.md

```diff
@@ -88,11 +89,11 @@ spec:
 Note that this is exactly the same way we would do a [canary deployment](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#canary-deployments) using plain Kubernetes, but in that case we would need to adjust the number of replicas of each Deployment to control the distribution of traffic. For example, to send 10% of the traffic to the canary version (**v2**), the replicas for **v1** and **v2** could be set to 9 and 1, respectively.
 
 However, since we are going to deploy the service in an [Istio enabled](/docs/setup/) cluster, all we need to do is set a routing
-rule to control the traffic distribution. For example if we want to send 10% of the traffic to the canary, we could use the
-[istioctl](/docs/reference/commands/istioctl/) command to set a routing rule something like this:
+rule to control the traffic distribution. For example if we want to send 10% of the traffic to the canary, we could use kubectl
```

Comment:
backticks around kubectl

Reply:
While you're doing this kind of stuff, would you mind taking kubectl and istioctl out of the .spelling file, and then putting backticks around any usage of those words as reported by the lint tool?

Note that you can run the linters locally using "make lint"